kustomize/commands/edit/add/addcomponent.go

```diff
@@ -5,9 +5,8 @@ package add
 
 import (
 	"errors"
-	"log"
-
 	"github.com/spf13/cobra"
+	"log"
```

Comment:
the presubmit check is failing because the imports here have been reordered; please sort them back to the way they were originally

Reply:
Thanks, GoLand IDE reordered them for some reason. I will revert them back right now.